src/game/logic/object/object.cpp

```diff
@@ -286,6 +287,29 @@ void Object::On_Collide(Object *other, const Coord3D *loc, const Coord3D *normal
     }
 }
 
+ExitInterface *Object::Get_Object_Exit_Interface() const
+{
+    ExitInterface *exit = nullptr;
+
+    for (BehaviorModule **module = m_allModules; *module != nullptr; module++) {
```

Comment:
I'd use a range based for loop here or `std::find`

Reply:
I prefer not to use range-based for, makes it harder to compare to the original pseudo.